packages/tsconfig.json

```diff
@@ -0,0 +1,109 @@
+{
```

Comment:
why this file is inside fetch-user?

Reply:
While starting jest I was working inside the fetch-user folder separately. It seems that I missed to delete the file.